pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -347,7 +364,7 @@ pub mod pallet {
 				&pool_id,
 				Some(PoolDetails::new(
 					who,
-					checked_curve,
+					checked_curve.clone(),
```

Comment:
what has changed? why does this need to be cloned now?

Reply:
Ah, yes, sorry. Was an artifact: [3b8b198](https://github.com/KILTprotocol/kilt-node/pull/789/commits/3b8b198cf9ec9ba04be197485bbbbe2a94b591c9)